NineChronicles.Headless.Tests/GraphTypes/TransactionHeadlessQueryTest.cs

```diff
@@ -316,8 +326,91 @@ private Task<ExecutionResult> ExecuteAsync(string query)
             return GraphQLTestUtils.ExecuteQueryAsync<TransactionHeadlessQuery>(query, standaloneContext: new StandaloneContext
             {
                 BlockChain = _blockChain,
-                Store = _store
+                Store = _store,
+                NineChroniclesNodeService = _service
             });
-        }  
-    }     
+        }
+
+        [Fact]
+        public async Task SignTransaction()
```

Comment:
It's not an important part. Is it intended as an E2E test, right?
Later, we should discuss the E2E tests convention (like all E2E should be named like `*ScenarioTest` or something..)